functions-and-scope.js

```diff
@@ -72,6 +124,18 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 9
 
+function highestGrade(list3) {
+    let highest = list3[0];
```

Comment:
Super dat je met `list3[0]` bent begonnen in plaats van `0`. Het is een veelgemaakte fout, dus lekker bezig! 🏅